### DISCUSSION TITLE
Cannot update Alignment Marks because of device is busy

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [jomjol/AI-on-the-edge-device#2439] [Q&A] [answered]
This is a fresh install and the first initialisazion. When I try to set the Alignment Marks this error message comes up:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/67194495/adfb4002-3206-4d42-9edd-aef5ae25675b)

I've already tried waiting for at least 5 minutes. Still the same.

**Comment by caco3** [comment]
Which version are you using?

What does the Overview page show?

Can you provide the log?

**Comment by 1Euro7Cent** [comment, marked as answer]
I actually fixed it. I gave the esp a static IP. that IP was already taken by home assistant. At the time the web itself was extremely unresponsive and 90% of the time wouldn't load at all. After I switch the static IP to an IP that wasn't taken it worked fine.
